src/en/clouds.md

```diff
@@ -39,6 +39,19 @@ rackspace    rackspace   DFW, ORD, IAD, LON, SYD, HKG
 
 This lists the cloud name (which you will use to specify the cloud you want to 
 use), its type (the API used to control it) and the known regions for the cloud.
+The default region for a cloud is shown first in the list-clouds output, so
+in the above, `us-east-1` is the default region for an aws cloud. To specify
+a different region, see [Creating a controller](./controllers-creating.html).
```

Comment:
I hear what you are saying. I don't feel strongly either way, although my original thought in placing it where I did is that it creates a nice segue from "here is how to find your default region" to "set the default region to something else". Putting "use a region other than the default" either between those two or after makes sense to me. Hmm. @evilnick , do you have a preference?